modules/@apostrophecms/asset/index.js

```diff
@@ -89,6 +89,7 @@ module.exports = {
 
   async init(self) {
 
+    self.options.devicePreviewMode = self.options.breakpointPreviewMode || self.options.devicePreviewMode;
```

Comment:
This is temporary until the rest of the logic can be changed over, I didn't want to cowboy that and risk bugs, the idea is to document `breakpointPreviewMode` and change the rest of the code when we have time

Reply:
I'm doing the changes, releasing with the two names will only make our life harder.